From: SJ Park <sj@kernel.org>
Date: Sun, 16 Aug 2026 13:38:09 -0700
Subject: [PATCH] ==== unurgent fixes 2 ====

mm/damon: fix wrong behaviors in DAMOS quota goals and sysfs refresh_ms

DAMOS quota goals and DAMON sysfs refresh_ms file have bugs that can
produce non critical but still unexpected behaviors.  Fix the two bugs.

Patches 1 and 2 fix the bugs in DAMOS PSI goal initialization and
eligible_mem_bp online commit, respectively.  Patch 3 fixes the bug in
DAMON sysfs refresh_ms file handling.

Changes from RFC v1.1
- RFC v1.1: https://lore.kernel.org/20260827134429.123023-1-sj@kernel.org
- None.
Changes from RFC
- RFC: https://lore.kernel.org/20260827042920.93580-1-sj@kernel.org
  (Mistakenly missed RFC tag)
- Fix build error.
- Add eligible_mem_bp nid commit fix.

Signed-off-by: SJ Park <sj@kernel.org>
---
 hkml_cv_bogus/hkml_cv_bogus_3p4tes70 | 0
 1 file changed, 0 insertions(+), 0 deletions(-)
 create mode 100644 hkml_cv_bogus/hkml_cv_bogus_3p4tes70

diff --git a/hkml_cv_bogus/hkml_cv_bogus_3p4tes70 b/hkml_cv_bogus/hkml_cv_bogus_3p4tes70
new file mode 100644
index 0000000000000..e69de29bb2d1d
-- 
2.47.3

